### PR TITLE
create tables for forum weekly digest data

### PR DESCRIPTION
**Pull request by s-cheng**
This PR is related to https://github.com/ForumMagnum/ForumMagnum/pull/7403. In an effort to make that PR more readable, I'm splitting out the code that sets up the two tables. Also running this on prod will let me (and whoever reviews that PR!) test the digest planner with real data locally.

The general idea is that we have Digests (that correspond to the forum's weekly email digest) and we have DigestPosts (that contain data about whether a post is in the email and/or on-site version of a digest). This migration sets up a dummy digest to make the planner easier to work with.

Future work, not part of this PR or https://github.com/ForumMagnum/ForumMagnum/pull/7403:
1. Importing past digest data
2. A page that displays the on-site version of the weekly digest (might do this right after)
3. Letting admins edit a digest's num/startDate/endDate

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204828394125025) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/lib/collections/digestPosts/schema.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { foreignKeyField } from '../../utils/schemaUtils'
+import { schemaDefaultValue } from '../../collectionUtils';
+
+export const DIGEST_STATUSES = ['yes', 'maybe', 'no'] as const
+export type InDigestStatuses = typeof DIGEST_STATUSES
+export type InDigestStatus = InDigestStatuses[number]
```
Comment (Will-Howard) [inline packages/lesswrong/lib/collections/digestPosts/schema.ts:6]: We have this TupleSet pattern for this situation (there is a long comment by the definition explaining why it's better)
```suggestion
const DIGEST_STATUSES = new TupleSet(['yes', 'maybe', 'no'] as const);
export type ReviewSortOption = UnionOf<typeof DIGEST_STATUSES>;
```

File: packages/lesswrong/lib/collections/digestPosts/schema.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { foreignKeyField } from '../../utils/schemaUtils'
+import { schemaDefaultValue } from '../../collectionUtils';
```
Comment (Will-Howard) [inline packages/lesswrong/lib/collections/digestPosts/schema.ts:2]: unused?

File: packages/lesswrong/lib/collections/digests/helpers.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import moment from "moment"
+import { combineUrls, getSiteUrl } from "../../vulcan-lib"
+
+/**
+ * Returns the digest name in our standard format
+ */
+export const getDigestName = ({digest, includeDates=true}: {digest: DigestsMinimumInfo, includeDates?: boolean}) => {
+  let name = `EA Forum Digest #${digest.num}`
```
Comment (Will-Howard) [inline packages/lesswrong/lib/collections/digests/helpers.ts:8]:
```suggestion
  const name = `EA Forum Digest #${digest.num}`
```

**Review by Will-Howard — APPROVED** [review]
LGTM, none of my comments are blocking

File: packages/lesswrong/lib/collections/digestPosts/schema.ts (hunk 1 above)
Comment (s-cheng) [reply packages/lesswrong/lib/collections/digestPosts/schema.ts:6]: Oh nice - appreciate this, thanks!

**Pull request merged**